src/lib.rs

```diff
@@ -1178,12 +1092,6 @@ mod panic {
 
 struct LoggerGuard(&'static Log);
```

Comment:
This type can be removed entirely now that we don't need a destructor, but that can be handled later.